### DISCUSSION TITLE
Map Not Showing All NavSatFix Points

### DISCUSSION THREAD

**Discussion by engnfrc** [foxglove/community#775] [Q&A] [answered]
Hello,

I'm using FoxGlove Studio for Desktop to plot a NavSatFix topic from an Inertial Navigation System and noticing that not all points are being shown. In particular, it looks like points are hidden when the system switches to Status = No Fix and is dead-reckoning and still estimating good latitude/longitude data. Is this expected behavior? Is there a way to prevent this filtering out? Based on some pages I was expecting the Map to still show the points, but modify how they look to indicate No Fix.

Thanks!

Josh Q

**Comment by jtbandes** [comment]
Yes, it appears we show no-fix messages in a different color:

https://github.com/foxglove/studio/blob/c64b5cca9d0c8ef81fbb85d9fca7cb8730f82e6b/packages/studio-base/src/panels/Map/MapPanel.tsx#L552-L568

Could you share sample data or screenshots to reproduce the issue?

**Comment by engnfrc** [comment]
Here's a sample bag file with the topic in question along with a snapshot of what I'm seeing. 

[sample.zip](https://github.com/foxglove/community/files/12090594/sample.zip)
![Screenshot 2023-07-18 224700](https://github.com/foxglove/community/assets/42557520/38bbb6a9-820b-4569-a5f1-6fb92eb3a4eb)

**Reply by jtbandes** [reply]
Thanks for the sample data, I agree this looks like a bug. Tracking internally: https://linear.app/foxglove/issue/FG-4364/map-not-showing-all-navsatfix-points

**Comment by bryfox** [comment, marked as answer]
This should be fixed in the latest version (v1.64.0) with https://github.com/foxglove/studio/pull/6516. If you're still seeing an issue with your data, please feel free to re-open this. Thanks for reporting.
